docs/tutorial/kymotracking.rst

```diff
@@ -123,38 +211,55 @@ an acceptable result.
 Maximum Likelihood Estimation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The function :func:`~lumicks.pylake.refine_tracks_gaussian()` instead uses an MLE optimization of a Poisson likelihood with a Gaussian expectation
-to characterize both the expected peak shape and photonic noise of the observed signal, adapted from :cite:`mortensen2010gauloc`.
-For each frame in the kymograph, we fit a small region around the tracked peak to the data by maximizing the following likelihood function:
+The function :func:`~lumicks.pylake.refine_tracks_gaussian()` instead uses a Maximum Likelihood Estimation (MLE) optimization of a Poisson
+likelihood with a Gaussian expectation to characterize both the expected peak shape and photonic noise of the observed signal (adapted for 1D data
+from :cite:`mortensen2010gauloc`).
+
+For each frame in the kymograph, we fit a small region around the tracked peak to the data by maximizing
+the following likelihood function:
 
 .. math::
 
     \mathcal{L(\theta)} = \prod_i^M e^{-E_i(\theta)} \frac{E_i(\theta)^{n_i}}{n_i!}
 
-where :math:`\theta` represents the parameters to be fitted, :math:`M` is the number of pixels to be fit and :math:`n_i` and :math:`E_i(\theta)`
+where :math:`\theta` represents the parameters to be fitted, :math:`M` is the number of pixels and :math:`n_i` and :math:`E_i(\theta)`
 are the observed photon count and expectation value for pixel :math:`i`. The shape of the peak is described with a Gaussian expectation function
 
 .. math::
 
     E_i(\theta) = \frac{N a}{\sqrt{2 \pi \sigma^2}} \exp \left[ \frac{-(x_i-\mu)^2}{2 \sigma^2} \right] + b
 
-Here :math:`N` is the total photons emitted in the fitted image (line), :math:`a` is the pixel size, :math:`\mu` is the peak center,
+Here :math:`N` is the total photons emitted in the fitted image (scan line), :math:`a` is the pixel size, :math:`\mu` is the peak center,
 :math:`x_i` is the pixel center position, :math:`\sigma^2` is the variance, and :math:`b` is the background level in
 photons/pixel.
 
-This function is called in a similar manner as the centroid refinement::
+This function is called in a similar manner as the centroid refinement. Since the MLE optimization is significantly slower than the centroid
+method, let's refine just a single long track::
+
+    # track a subsection of the kymo
+    cropped_kymo = kymo40.crop_by_distance(4.2, 5.8)
+    tracks = lk.track_greedy(cropped_kymo, "green", track_width=0.3, window=10, pixel_threshold=9)
+    tracks = lk.filter_tracks(tracks, minimum_length=10)
+
+    # perform the gaussian refinement
+    refined = lk.refine_tracks_gaussian(tracks, window=3, refine_missing_frames=False, overlap_strategy="skip")
```

Comment:
for the first comment, I think the paragraph overall reads better in the order it is now. I also would hope that `"multiple"` isn't used in most cases; ideally the experiment should be performed such that you get well separated tracks (although I get that this can't always be the case). Maybe though we should mention that this is the goal since very congested kymos are just difficult to analyze properly anyway (increased chance of track crossing, merging, etc. which our algorithm can't handle anyway)?

Reply:
In principle true, but one can likely get a better separation between offset and Gaussian tail if you use a bigger window. Allowing overlap would allow you to increase the window size and include a bigger chunk to get a better estimate of the background (even though the peaks would be distinct).

But for now, I'm ok with leaving the order as is, since we haven't done much testing to assess how big an effect this would have on bias/variance.